tests/r/FormatRBearTest.py

```diff
@@ -129,3 +138,9 @@
     valid_files=(good_file_6,),
     invalid_files=(bad_file_3,),
     settings={'r_max_expression_length': '25'})
+
+FormatRBearRUseBothArrowAndEqualTest = verify_local_bear(
```

Comment:
up to you:
Maybe call it `FormatRBearDefaultSettingsTest`? Then you can test everything regarding the default setup there :)